src/components/DeleteItemModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import DefaultModal from './DefaultModal';
+import Logo from '../molecules/Logo';
+
+const Desc = styled.p`
+	color: black;
```

Comment:
Is the color not black by default?

Reply:
default color is white for some reason :o